src/coreclr/jit/flowgraph.cpp

```diff
@@ -882,10 +882,12 @@ GenTreeCall* Compiler::fgGetStaticsCCtorHelper(CORINFO_CLASS_HANDLE cls, CorInfo
     size_t   moduleID;
     void*    pclsID;
     void*    pmoduleID;
+    int      dataBlobOffset;
+    int      initFlag;
 
     clsID = info.compCompHnd->getClassDomainID(cls, &pclsID);
```

Comment:
We should merge the `getClassDomainID` and `getClassModuleIdForStatics` JIT/EE interface methods together, and change the merged method to return struct with all details required to emit the Cctor.